source/extensions/retry/priority/other_priority/other_priority.cc

```diff
@@ -55,6 +52,13 @@ void OtherPriorityRetryPriority::adjustForAttemptedPriorities(
   const auto& adjusted_per_priority_health = adjustedHealthAndSum.first;
   auto total_health = adjustedHealthAndSum.second;
 
+  // If total health is still zero at this point, it must mean that the original cluster is
```

Comment:
Optional nit
s/the original cluster/all clusters/?
Both statements are true but I think the latter would be a bit more informative.